content/authorization.md

```diff
@@ -382,6 +383,20 @@ where the audience is provided:
 &aud=https%3A%2F%2Ffhir-ehr.sandboxcerner.com%2Fdstu2%2F0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca%2F
 </pre>
 
+#### Redirect URI ####
+
+When performaing an authorization grant request the application may provide a redirect uri. 
```

Comment:
performaing -> performing

We should make it clear that in situation where there are multiple registered redirect URIs, they must provide a `redirect_uri`.

Reply:
@kolkheang  Wouldnt using  ```must``` imply that its required for the multiple redirect uri scenarios ?
Since its optional hence I choose ```may provide a redirect_uri```

Reply:
Fixed the typo here : https://github.com/cerner/fhir.cerner.com/pull/370/commits/f833a4f6ab9a21f7028085efb35ab19ef0a578b7

Still waiting on thoughts around using ```must``` as compared to ```may```.

Reply:
I just re-read the wording in this section and what you have is fine with regard to the wording of may vs must.

👍